docs/master/api-reference/directives.md

```diff
@@ -2156,6 +2178,64 @@ type PostEdge {
 }
 ```
 
+### Simple Pagination
+
+The default `PAGINATOR` uses the query builder method `paginate` which will
```

Comment:
Is there a better place for this explanatory text? I find it a bit confusing to have this whole paragraph about another pagination method as the first thing to read under `Simple Pagination`.

Reply:
```suggestion
In contrast to other pagination types, `SIMPLE` pagination only fires a single database
query on every request. This improves performance, but means that the response does not
hold information about the total number of items.
```

Reply:
Oh yes, that's true. Very good 👍